tls/s2n_signature_scheme.c

```diff
@@ -339,3 +339,20 @@ const struct s2n_signature_preferences s2n_certificate_signature_preferences_202
     .count = s2n_array_len(s2n_sig_scheme_pref_list_20201110),
     .signature_schemes = s2n_sig_scheme_pref_list_20201110,
 };
+
+/* Based on s2n_sig_scheme_pref_list_20140601 but with all hashes < SHA-384 removed */
+const struct s2n_signature_scheme* const s2n_sig_scheme_pref_list_20210816[] = {
+        /* RSA PKCS1 */
+        &s2n_rsa_pkcs1_sha384,
+        &s2n_rsa_pkcs1_sha512,
+
+        /* ECDSA - TLS 1.2 */
```

Comment:
Might want to drop the TLS1.2 part of this comment. 1/3 of your schemes here is TLS1.3 :)

But if not including any TLS1.3 cipher suites is intentional, you could also just drop s2n_ecdsa_secp384r1_sha384.